src/ontology/index.js

```diff
@@ -154,13 +154,7 @@ const uploadFromJSON = async ({ data, conn }) => {
 
     try {
         await Promise.all(Object.entries(sources).map(async ([sourceKey, sourceDefn]) => {
-            const sourceRID = rid(await conn.addRecord({
-                content: sourceDefn,
-                existsOk: true,
-                fetchConditions: { name: sourceDefn.name },
-                target: 'Source',
-                upsert: true,
-            }));
+            const sourceRID = rid(await conn.addSource(sourceDefn));
```

Comment:
fetchConditions and upsert aren't provided by addRecord - are they not needed here?

Reply:
sorry, just upsert

Reply:
good catch! i think upsert must have been lost in a merge or something. I will re-add it

Reply:
fixed